src/utils/algos.js

```diff
@@ -0,0 +1,15 @@
+export function equalsArray(a, b) {
+    if (!b || !a) return false;
+    if (b === a) return true;
```

Comment:
if b and a are the same number or string or Object (well anything instead of Array) you could return true ?
Maybe the first thing to check is : 
`Array.isArray(a) && Array.isArray(b)`